.github/workflows/pythonpublish.yml

```diff
@@ -227,3 +227,45 @@ jobs:
           file: ./plugins/flytekit-spark/Dockerfile
           cache-from: type=gha
           cache-to: type=gha,mode=max
+
+  build-and-push-flyin-images:
+    runs-on: ubuntu-latest
+    needs: deploy
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: "0"
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to GitHub Container Registry
+        if: ${{ github.event_name == 'release' }}
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: "${{ secrets.FLYTE_BOT_USERNAME }}"
+          password: "${{ secrets.FLYTE_BOT_PAT }}"
+      - name: Prepare flyin Image Names
+        id: flyin-names
+        uses: docker/metadata-action@v3
+        with:
+          images: |
+            ghcr.io/${{ github.repository_owner }}/flytekit
+          tags: |
+            flyin-latest
+            flyin-${{ github.sha }}
+            flyin-${{ needs.deploy.outputs.version }}
+      - name: Push Flyin Image to GitHub Registry
+        uses: docker/build-push-action@v2
+        with:
+          context: "./plugins/flytekit-flyin/"
+          platforms: linux/arm64, linux/amd64
+          push: ${{ github.event_name == 'release' }}
+          tags: ${{ steps.flyin-names.outputs.tags }}
+          build-args: |
+            VERSION=${{ needs.deploy.outputs.version }}
```

Comment:
we missed the python version here (like [this](https://github.com/flyteorg/flytekit/blob/master/.github/workflows/pythonpublish.yml#L142)). Do we need to have multiple versions of python in this image?

Reply:
This is causing an issue in releases: https://github.com/flyteorg/flytekit/actions/runs/7108516439/workflow#L142

Reply:
no problem, thanks a lot!